pkg/cloudprovider/fake/instancetype.go

```diff
@@ -23,24 +23,24 @@ import (
 	"github.com/samber/lo"
 	"k8s.io/apimachinery/pkg/util/sets"
 
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 
-	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	v1 "sigs.k8s.io/karpenter/pkg/apis/v1"
 	"sigs.k8s.io/karpenter/pkg/cloudprovider"
 	"sigs.k8s.io/karpenter/pkg/scheduling"
 )
 
 const (
-	LabelInstanceSize                       = "size"
-	ExoticInstanceLabelKey                  = "special"
-	IntegerInstanceLabelKey                 = "integer"
-	ResourceGPUVendorA      v1.ResourceName = "fake.com/vendor-a"
-	ResourceGPUVendorB      v1.ResourceName = "fake.com/vendor-b"
+	LabelInstanceSize                           = "size"
+	ExoticInstanceLabelKey                      = "special"
+	IntegerInstanceLabelKey                     = "integer"
+	ResourceGPUVendorA      corev1.ResourceName = "fake.com/vendor-a"
+	ResourceGPUVendorB      corev1.ResourceName = "fake.com/vendor-b"
```

Comment:
If you want nicer indenting, you can do 
```suggestion
	ResourceGPUVendorA  = corev1.ResourceName("fake.com/vendor-a")
	ResourceGPUVendorB = corev1.ResourceName("fake.com/vendor-b")
```